src/pages/ProfilePage.spec.js

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/react';
+import ProfilePage from './ProfilePage';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('ProfilePage', () => {
+  it('renders a headline and an exit-button', () => {
+    render(
+      <MemoryRouter>
+        <ProfilePage />
+      </MemoryRouter>
+    );
+
+    const headline = screen.getByText('Mein Katzenprofil');
+    const exitButton = screen.getByTestId('button to leave the app');
+
+    expect(headline).toBeInTheDocument();
+    expect(exitButton).toBeInTheDocument();
+  });
+
+  it('renders an bin-icon and an update-icon', () => {
+    render(
+      <MemoryRouter>
+        <ProfilePage />
+      </MemoryRouter>
+    );
+
+    const binicon = screen.getByAltText('bin icon');
+    const updateicon = screen.getByAltText('icon rotate arrow to the left');
+
+    expect(binicon).toBeInTheDocument();
+    expect(updateicon).toBeInTheDocument();
+  });
+
+  it('renders 3 input-Elements for text, 1 input-Element for file and one textarea', () => {
```

Comment:
If your write the whole sentence in lower case the `Element` should be lower case as well :)

Reply:
yes thanks, you are right - i will fix it